src/widgets/SimpleTable.js

```diff
@@ -38,12 +39,17 @@ export const SimpleTable = React.memo(
         rowData => {
           const { id } = rowData;
 
-          return columns.map(({ key, width, alignText }, index) => {
+          return columns.map(({ key, width, alignText, type }, index) => {
             const disabledStyle = disabledRows?.[id] || isDisabled ? { color: GREY } : {};
             const textStyle = { ...cellText[alignText], ...disabledStyle };
+            const valueText =
+              type === 'date'
+                ? rowData[key]?.toDateString() ?? generalStrings.not_available
```

Comment:
Believe it's the same format but can we use this `formatDate` method used here: https://github.com/openmsupply/mobile/blob/311babb8092fb578a062364c1be18e950ec07c4b/src/widgets/DataTable/DataTableRow.js#L282

Reply:
Cool format looks slightly different but I think the formatDate version makes more sense - committed it with e9f640d9:
- toDateString() => Wed Jul 29 1998
- formatDate() => 29/7/1998